crates/database/src/lib.rs

```diff
@@ -49,42 +50,60 @@ use {
 pub type PgTransaction<'a> = sqlx::Transaction<'a, sqlx::Postgres>;
 
 /// The names of tables we use in the db.
-pub const TABLES: &[&str] = &[
-    "orders",
-    "trades",
-    "invalidations",
-    "last_indexed_blocks",
-    "quotes",
-    "settlements",
-    "presignature_events",
-    "order_quotes",
-    "solver_competitions",
-    "auctions",
-    "competition_auctions",
-    "onchain_placed_orders",
-    "ethflow_orders",
-    "order_execution",
-    "interactions",
-    "ethflow_refunds",
-    "settlement_scores",
-    "settlement_observations",
-    "auction_prices",
-    "auction_participants",
-    "app_data",
-    "jit_orders",
-];
+static TABLES: OnceCell<&'static [&'static str]> = OnceCell::const_new();
```

Comment:
Why not use owned strings here to avoid having to leak them?
```suggestion
static TABLES: OnceCell<Vec<String>> = OnceCell::const_new();
```